src/main/drivers/bus_i2c_hal.c

```diff
@@ -46,8 +46,8 @@ const i2cHardware_t i2cHardware[I2CDEV_COUNT] = {
     {
         .device = I2CDEV_1,
         .reg = I2C1,
-        .sclPins = { DEFIO_TAG_E(PB6), DEFIO_TAG_E(PB8) },
-        .sdaPins = { DEFIO_TAG_E(PB7), DEFIO_TAG_E(PB9) },
+        .sclPins = { { DEFIO_TAG_E(PB6) }, { DEFIO_TAG_E(PB8) } },
```

Comment:
It would be better to use macro here too:

(in common file):
```
#ifdef STM32F4
# define I2CPINDEF(pin, af) { DEFIO_TAG_E(pin), af }
#else
# define I2CPINDEF(pin) { DEFIO_TAG_E(pin) }
#endif